s4cmd.py

```diff
@@ -830,6 +830,16 @@ def update_privilege(self, obj, target):
       os.chmod(target, int(obj['Metadata']['privilege'], 8))
 
   @log_calls
+  def print_files(self, source):
+    '''Print out a series of files'''
+    sources = self.source_expand(source)
+
+    for source in sources:
+      s3url = S3URL(source)
+      response = self.s3.get_object(Bucket=s3url.bucket, Key=s3url.path)
+      message(response['Body'].read())
```

Comment:
Thanks for doing this. This will break if the content contains '%' character. It should be

  message('%s', response['Body'].read())

Also, would this work for unicode characters correctly?